app/models/manageiq/providers/ansible_tower/automation_manager/machine_credential.rb

```diff
@@ -1,2 +1,66 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential < ManageIQ::Providers::AutomationManager::Authentication
+  FIELDS = {
+    :description => {
+      :type       => :string,
+      :required   => false,
+      :label      => _('Description'),
+      :help_text  => _('Optional description of this credential.'),
+      :encrypted  => false
+    },
+    :username         => {
+      :type       => :string,
+      :required   => false,
+      :label      => _('Username'),
+      :help_text  => _('Username for this credential.'),
+      :max_length => 1024,
+      :encrypted  => false
+    },
+    :ssh_key_data     => {
+      :type       => :string,
+      :required   => false,
+      :label      => _('SSH private key'),
+      :help_text  => _('RSA or DSA private key to be used instead of password.'),
+      :encrypted  => true
+    },
+    :ssh_key_unlock   => {
+      :type       => :string,
+      :required   => false,
+      :label      => _('SSH key unlock'),
+      :help_text  => _('Passphrase to unlock SSH private key if encrypted (or \'ASK\' to prompt the user for machine credentials).'),
+      :max_length => 1024,
+      :encrypted  => true
+    },
+    :become_method    => {
+      :type       => :choice,
```

Comment:
Ah...choice is a strange name choice.  In HTML they call this the `:select` tag (with an entry called an `:option`)...in many languages, it's an `:enum` (although `:enum` doesn't imply a single selection).  I'm also trying to think what the analagous name would be when multiple selections are allowed...typically those are called `:multi_select`, which is why I think `:select` might be best?  @blomquisg Thoughts?